stubs/PyAutoGUI/pyautogui/__init__.pyi

```diff
@@ -0,0 +1,230 @@
+    interval: float = ...,
+    button: str = ...,
+    duration: float = ...,
+    tween: Callable[[float], float] = ...,
+    logScreenshot: bool | None = ...,
+    _pause: bool = ...,
+) -> None: ...
+def tripleClick(
+    x: float | Sequence[int] | None = ...,
+    y: float | Sequence[int] | None = ...,
+    interval: float = ...,
+    button: str = ...,
+    duration: float = ...,
+    tween: Callable[[float], float] = ...,
+    logScreenshot: bool | None = ...,
+    _pause: bool = ...,
+) -> None: ...
+def scroll(
+    clicks: float,
+    x: float | Sequence[int] | None = ...,
+    y: float | Sequence[int] | None = ...,
+    logScreenshot: bool | None = ...,
+    _pause: bool = ...,
+) -> None: ...
+def hscroll(
+    clicks: float,
+    x: float | Sequence[int] | None = ...,
+    y: float | Sequence[int] | None = ...,
+    logScreenshot: bool | None = ...,
+    _pause: bool = ...,
+) -> None: ...
+def vscroll(
+    clicks: float,
+    x: float | Sequence[int] | None = ...,
+    y: float | Sequence[int] | None = ...,
+    logScreenshot: bool | None = ...,
+    _pause: bool = ...,
+) -> None: ...
+def moveTo(
+    x: float | Sequence[int] | None = ...,
+    y: float | Sequence[int] | None = ...,
+    duration: float = ...,
+    tween: Callable[[float], float] = ...,
+    logScreenshot: bool = ...,
+    _pause: bool = ...,
+) -> None: ...
+def moveRel(
+    xOffset: float | Sequence[int] | None = ...,
+    yOffset: float | Sequence[int] | None = ...,
+    duration: float = ...,
+    tween: Callable[[float], float] = ...,
+    logScreenshot: bool = ...,
+    _pause: bool = ...,
+) -> None: ...
+
+move = moveRel
+
+def dragTo(
+    x: float | Sequence[int] | None = ...,
+    y: float | Sequence[int] | None = ...,
+    duration: float = ...,
+    tween: Callable[[float], float] = ...,
+    button: str = ...,
+    logScreenshot: bool | None = ...,
+    _pause: bool = ...,
+    mouseDownUp: bool = ...,
+) -> None: ...
+def dragRel(
+    xOffset: float | Sequence[int] = ...,
+    yOffset: float | Sequence[int] = ...,
+    duration: float = ...,
+    tween: Callable[[float], float] = ...,
+    button: str = ...,
+    logScreenshot: bool | None = ...,
+    _pause: bool = ...,
+    mouseDownUp: bool = ...,
+) -> None: ...
+
+drag = dragRel
+
+def isValidKey(key: str) -> bool: ...
+def keyDown(key: str, logScreenshot: bool | None = ..., _pause: bool = ...) -> None: ...
+def keyUp(key: str, logScreenshot: bool | None = ..., _pause: bool = ...) -> None: ...
+def press(
+    keys: str | list[str], presses: int = ..., interval: float = ..., logScreenshot: bool | None = ..., _pause: bool = ...
+) -> None: ...
+def hold(keys: str | list[str], logScreenshot: bool | None = ..., _pause: bool = ...) -> Generator[None, None, None]: ...
+def typewrite(message: str | list[str], interval: float = ..., logScreenshot: bool | None = ..., _pause: bool = ...) -> None: ...
+
+write = typewrite
+
+def hotkey(*args: str, **kwargs: float | bool | None) -> None: ...
```

Comment:
At runtime, the signature is `*args, **kwargs`, but if we do `*args, **kwargs` in the stub, type checkers won't emit an error if somebody has a typo in their code like `logScreenshot=Falsse`. The only kwargs that would actually be useful here are `logScreenshot` and `interval`:

```suggestion
def hotkey(*args: str, logScreenshot: bool | None = ..., interval: float = ...) -> None: ...
```

Reply:
I wasn't sure if I should be expanding the kwargs or not. Good to know in cases of specifically known kwargs I'll define them!